lib/mayaHydra/mayaPlugin/viewCommand.cpp

```diff
@@ -79,10 +69,22 @@ constexpr auto _visibleOnlyLong = "-visibleOnly";
 constexpr auto _sceneDelegateId = "-sid";
 constexpr auto _sceneDelegateIdLong = "-sceneDelegateId";
 
-// MAYA-127221: We will need to replace this with a flag inside of the pluginInfo command in an
-// upcoming release.
-constexpr auto _pluginInfoCutId = "-cid";
-constexpr auto _pluginInfoCutIdLong = "-pluginInfoCut";
+// Versioning and build information.
+constexpr auto _majorVersion = "-mjv";
```

Comment:
Core file in the review: add flags to the mayaHydra command to get version and build information.